bin/deploy.sh

```diff
@@ -131,33 +90,29 @@ mv wp-cli.phar /usr/local/bin/wp
 runuser -l vagrant -c "curl -O https://raw.githubusercontent.com/wp-cli/wp-cli/v2.4.0/utils/wp-completion.bash && mv wp-completion.bash ~/.bash_completion"
 
 echo "Configure services and cronjobs"
-cp /data/www/uploader/scripts/uploader.service /lib/systemd/system/
+rm /etc/systemd/system/uploader.service
+ln -s /data/www/uploader/scripts/uploader.service /etc/systemd/system/
```

Comment:
Ah, removing `bootstrap.sh` made GitHub display this diff more usefully!

+1 to symlinking rather than copying. However, we should either `rm -f` to avoid spurious missing file errors, or `ln --force` to skip the `rm` entirely.